### PR TITLE
Update for RDoc 3.12 and Ruby 1.9.3

### PR DESCRIPTION
**Pull request by Azolo**
Updated for [Luis's request](%28https://groups.google.com/forum/?fromgroups#!topic/rubyinstaller/T99WwBamgdY%29) in the rubyinstaller group.

Mainly just updates to get it working with RDoc 3.12

To use the rubyinstaller `docs` task to build the docs you have to change the code page to UTF-8 via `chcp 65001`

### CODE REVIEW

**Comment by Azolo** [comment]
Maybe not, File pages aren't actually getting created. The root of this comes a legacy template support bug in RDoc::Darkfish

Also the code colors on some of the pages don't show up well.
I need to look into that.

**Comment by Azolo** [comment]
Alright it looks like it's fixed now, but it uses the newer Darkfish css file.

File: lib/rdoc/generator/template/chm/classpage.rhtml [hunk 1]
```diff
@@ -182,23 +177,3 @@
...
-	<% end %>
-	</div>
-
-	<div id="validator-badges">
-		<p><small><a href="http://validator.w3.org/check/referer">[Validate]</a></small></p>
-		<p><small>Generated with the <a href="http://deveiate.org/projects/Darkfish-Rdoc/">Darkfish
-			Rdoc Generator</a> <%= RDoc::Generator::Darkfish::VERSION %></small>.</p>
-	</div>
-
-</body>
```
Comment (luislavena) [inline lib/rdoc/generator/template/chm/classpage.rhtml:202]: Haven't checked this yet, but are you sure about removing the `body` and `html` closing tags?

Comment (Azolo) [inline lib/rdoc/generator/template/chm/classpage.rhtml:202]: I in fact should not be.

I'll update the pull request.

Thanks

**Pull request merged**